### ISSUE TITLE
Buld built Jan 31 2023 @ 11:07:35, made What's new visible again.

### ISSUE THREAD

**Issue by vkovacs** [zero318/BegoneWhatsNew#8]
After the latest update "What's new" is visible again.

**Comment by ghost** [comment]
Yeah, doesn't work anymore

**Comment by swanknight** [comment]
Same issue here, Valve changed the CSS again for some stupid reason. 😠 
I found a workaround, I tested it, and it's working...for now.

> 1. Open the file steamui\css\chunk~547eb3232.css on your favorite text editor
>
> 2. Control+F the first occurrence of libraryhome_UpdatesContainer_17uEB, place your cursor after the { and press "Insert" so your typing overwrites what's in there (we want to keep the file size the same as before)
>
> 3. Type in display: none !important; and then press space until you reach the }. It should look [like this](https://i.imgur.com/ZROFOiG.png)

**Source:** https://old.reddit.com/r/Steam/comments/10r4ksg/whats_new_is_back_after_recent_update_despite/j6v407y/

**Comment by Kopert** [comment]
I'm the one who posted the guide above on Reddit. This was my old method of hiding the What's New section before using a Steam skin to do it. Right now, however, I can't get it to work as part of a skin anymore.

I tried placing the usual rules for `libraryhome_UpdatesContainer_17uEB` on my skin's webkit.css but nothing I tried worked. I've been following some other discussions around and, so far, couldn't find a working solution.

My workaround works for now, but it could be patched out, the class name might change in the future and the fix needs to be reapplied every time Steam updates the .css file.

**Comment by swanknight** [comment]
> I'm the one who posted the guide above on Reddit. This was my old method of hiding the What's New section before using a Steam skin to do it. Right now, however, I can't get it to work as part of a skin anymore.
> 
> I tried placing the usual rules for `libraryhome_UpdatesContainer_17uEB` on my skin's webkit.css but nothing I tried worked. I've been following some other discussions around and, so far, couldn't find a working solution.
> 
> My workaround works for now, but it could be patched out, the class name might change in the future and the fix needs to be reapplied every time Steam updates the .css file.

I tried to add this line to my skin's webkit.css file, and it works for about 30 seconds, then that "What's New" section pops up again like a curse.

```
.libraryhome_UpdatesContainer_17uEB {
    display: none !important;
}
```

**Comment by Kopert** [comment]
I had been doing some research on this. I'll post my findings below for future reference.

Apparently, "when Steam released the [new Steam Library update](https://store.steampowered.com/libraryupdate), the library stopped following the Steam client theme" <sup>[[1]](https://github.com/PhantomGamers/SFP#library-skinning)</sup>. What this means is that the rules on the `webkit.css` file no longer apply to the Steam library, where the What's New section is located. There is some speculation that the library rendering has been moved to another process that no longer references the `webkit.css` file.

There is a [3rd party tool](https://github.com/PhantomGamers/SFP) that patches some steamui css files allowing for custom library skinning rules to be loaded from a `steamui/libraryroot.custom.css` file.

Basically what we're doing with our workaround is a "low-level" manual patch. If you wish to use the third party tool, you can then create the `libraryroot.custom.css` file and place this on it:

    .libraryhome_UpdatesContainer_17uEB {
        display: none !important;
    }

I haven't tested, but apparently this could also work and would be more resistant to changes:

    [class*="libraryhome_UpdatesContainer_"] {
        display: none !important;
    }

Personally, I'll stick to patching the chunk css manually, since I don't need anything else skinned on the library. If anyone has a cleaner solution to offer, please do, but for now it seems that it is not possible to achieve what we want by just dropping and using a skin anymore.

**Comment by sunlightspectre** [comment]
@Kopert Just so you know, I'll be trying out soft-patching steam using your second menu. I think it should work for now - as of 09-02-2023, but I'm fully expecting it to be made impossible by future updates within the span of the month. I don't like wading into other's peoples conversation, but I might stop by depending on how it goes with thoughts and feedback. Thanks for putting your own research together.

**Comment by straef** [comment]
> There is a [3rd party tool](https://github.com/PhantomGamers/SFP) that patches some steamui css files allowing for custom library skinning rules to be loaded from a `steamui/libraryroot.custom.css` file.
> 
> 
> I haven't tested, but apparently this could also work and would be more resistant to changes:
> 
> ```
> [class*="libraryhome_UpdatesContainer_"] {
>     display: none !important;
> }
> ```

i just did this and it is currently working. probably will not remember to report back after steam updates something. thank you for pointing me to this!

**Comment by PaperbackSpine** [comment]
Edit: nope I was wrong.

I thought using Family View would block "What's New", but it's even worse. "What's New" remains visible and even lets users circumvent the Family View restrictions.

**Comment by HazeyDays** [comment]
Fix:
https://github.com/zero318/BegoneWhatsNew/issues/9#issuecomment-1472882626

**Comment by swanknight** [comment]
> > There is a [3rd party tool](https://github.com/PhantomGamers/SFP) that patches some steamui css files allowing for custom library skinning rules to be loaded from a `steamui/libraryroot.custom.css` file.
> > I haven't tested, but apparently this could also work and would be more resistant to changes:
> > ```
> > [class*="libraryhome_UpdatesContainer_"] {
> >     display: none !important;
> > }
> > ```
> 
> i just did this and it is currently working. probably will not remember to report back after steam updates something. thank you for pointing me to this!

I just tested it, and this method works...for now. At least, it's the easiest method so far.
